src/dbpath.cc

```diff
@@ -18,15 +18,16 @@
  *
  */
 
-#include "fty-common/db/dbpath.h"
-#include "fty-common/log/fty_log.h"
+#include "fty_common_db_classes.h"
+//#include "fty-common/db/dbpath.h"
+//#include "fty-common/log/fty_log.h"
 
 #include <stdlib.h>
 #include <stdio.h>
 #include <fstream>
 #include <string.h>
 #include <stdlib.h>
-#include "fty-common/common/filesystem.h"
+#include <fty-common/common/filesystem.h>
```

Comment:
this is part of fty-common now and you are not linking it

Reply:
Thanks for noting that Bara, I intend to simplify fty-common to remove subdirectories, so this would be invalid after all. I'll update it soon.